kodilla-spring/src/main/java/com/kodilla/spring/library/Library.java

```diff
@@ -5,7 +5,7 @@
 import java.util.ArrayList;
 import java.util.List;
 
-@Service
+//@Service
```

Comment:
Nie wrzucamy do repozytorium zakomentowanego kodu. Proszę, skasuj to :-)

Reply:
Zmiany "zabrały" się z zadaniem. Poprawione.